src/script/components/list/ConversationListCallingCell.tsx

```diff
@@ -182,83 +183,109 @@ const ConversationListCallingCell: React.FC<CallingCellProps> = ({
+                  'conversation-list-cell-center-no-left': temporaryUserStyle,
+                })}
+              >
+                <span className="conversation-list-cell-name">{conversationName}</span>
+
+                {isIncoming && (
+                  <span className="conversation-list-cell-description" data-uie-name="call-label-incoming">
+                    {t('callStateIncoming')}
+                  </span>
+                )}
+                {isOutgoing && (
+                  <span className="conversation-list-cell-description" data-uie-name="call-label-outgoing">
+                    {t('callStateOutgoing')}
+                  </span>
+                )}
+                {isConnecting && (
+                  <span className="conversation-list-cell-description" data-uie-name="call-label-connecting">
+                    {t('callStateConnecting')}
+                  </span>
+                )}
+                {isOngoing && startedAt && (
+                  <div className="conversation-list-info-wrapper">
+                    <span
+                      className="conversation-list-cell-description"
+                      data-uie-name="call-duration"
+                      aria-label={t('callDurationLabel')}
+                    >
+                      <Duration {...{startedAt}} />
+                    </span>
+                    {isCbrEnabled && (
+                      <span className="conversation-list-cell-description" data-uie-name="call-cbr">
+                        {t('callStateCbr')}
+                      </span>
+                    )}
+                  </div>
+                )}
+              </div>
             </div>
             <div className="conversation-list-cell-right">
               {isConnecting ||
                 (isOngoing && (
-                  <div
+                  <button
                     className="call-ui__button call-ui__button--red"
                     onClick={() => callActions.leave(call)}
                     title={t('videoCallOverlayHangUp')}
+                    type="button"
                     data-uie-name="do-call-controls-call-leave"
                   >
                     <Icon.Hangup className="small-icon" style={{maxWidth: 17}} />
-                  </div>
+                  </button>
                 ))}
             </div>
           </div>
           {(isOngoing || selfHasActiveVideo) && isMinimized && !!videoGrid?.grid?.length ? (
             <div
               className="group-video__minimized-wrapper"
               onClick={isOngoing ? () => multitasking.isMinimized(false) : undefined}
+              onKeyDown={
+                isOngoing
+                  ? (event: React.KeyboardEvent<HTMLInputElement>) => {
+                      if (event.key === KEY.ENTER || event.key === KEY.SPACE) {
+                        multitasking.isMinimized(false);
+                      }
+                    }
+                  : undefined
+              }
```

Comment:
can we extract it to an event handler? inline function is re-created on every render